.husky/pre-commit

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# format code style
+yarn format
+
+# handle linting errors
+ yarn lint
```

Comment:
`yarn format` should (will) exit with a non-zero code when there are remaining issues.

```suggestion
```

Reply:
Isn't that a good thing? :slightly_smiling_face: